src/clarity-angular/tree-view/all.spec.ts

```diff
@@ -0,0 +1,12 @@
+/* * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import TreeNodeSpecs from "./tree-node.spec";
+import TreeViewSpecs from "./tree-view.spec";
+
+describe("Tree View Directives", () => {
+    TreeNodeSpecs();
+    TreeViewSpecs();
+});
```

Comment:
We seriously need a spec on AbstractTreeSelection. That's where 90% of the logic is.

It would also be good to have specs on the TreeSelectionService, since it apparently has a validation method.